PIL/Image.py

```diff
@@ -667,6 +715,7 @@ def __setstate__(self, state):
         self.frombytes(data)
 
     def tobytes(self, encoder_name="raw", *args):
```

Comment:
No, but the interface dates from that era, and there could be private image plugins out in the wild using the interface.  So we can't just change the interface and touch up our plugins, if we make changes, we need to do backwards compatibility and deprecation periods.